projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/secrets/service/vault/VaultJobSecretsService.java

```diff
@@ -27,7 +27,8 @@
 @ConditionalOnProperty(value = "featureflag.vault.integration.enabled")
 public class VaultJobSecretsService implements com.vmware.taurus.secrets.service.JobSecretsService {
 
-  private static final int VAULT_SIZE_LIMIT_DEFAULT = 1048576; // 1 MB
+  //  package private so it can be used in tests
```

Comment:
you can use the annotation @VisibleForTesting.

Reply:
I don't see this used anywhere else and I really don't want to make any more changes.